packages/mermaid/src/diagram-api/types.ts

```diff
@@ -26,7 +27,7 @@ export interface DiagramDefinition {
   db: DiagramDb;
   renderer: any;
   parser: any;
-  styles: any;
+  styles?: any;
```

Comment:
Make sure this change is propagated to places where styles is consumed.

Like [addStylesForDiagram(id, diagram.styles);](https://github.com/mermaid-js/mermaid/blob/af13f3430aa994e7dd258aa5964894c476dc00ac/packages/mermaid/src/diagram-api/diagramAPI.ts#L53-L54)

Reply:
Oh, I updated to see If it causes any issues but there was none so didn't pay much attention to that, I'll look into details soon.

Reply:
Try doing a search for `.styles`, there might be some useage in untyped files.